datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -233,6 +238,150 @@ type Query {
     dataPlatformInstance(urn: String!): DataPlatformInstance
 }
 
+"""
+A ERModelRelation is a high-level abstraction that dictates what datasets fields are ermodelrelationed.
+"""
+type ERModelRelation implements EntityWithRelationships & Entity {
+    """
+    The primary key of the role
+    """
+    urn: String!
+
+    """
+    The standard Entity Type
+    """
+    type: EntityType!
+
+    """
+    Unique id for the ermodelrelation
+    """
+    ermodelrelationId: String!
+
+    """
+    An additional set of read only properties
+    """
+    properties: ERModelRelationProperties
+
+    """
+    An additional set of of read write properties
+    """
+    editableProperties: ERModelRelationEditableProperties
+
+    """
+    References to internal resources related to the dataset
+    """
+    institutionalMemory: InstitutionalMemory
+
+    """
+    Ownership metadata of the dataset
+    """
+    ownership: Ownership
+
+    """
+    Status of the Dataset
+    """
+    status: Status
+
+    """
+    Tags used for searching dataset
+    """
+    tags: GlobalTags
+
+    """
+    The structured glossary terms associated with the dataset
+    """
+    glossaryTerms: GlossaryTerms
+
+    """
+    List of relationships between the source Entity and some destination entities with a given types
+    """
+    relationships(input: RelationshipsInput!): EntityRelationshipsResult
+
+    """
+    Privileges given to a user relevant to this entity
+    """
+    privileges: EntityPrivileges
+
+    """
+    No-op required for the model
+    """
+    lineage(input: LineageInput!): EntityLineageResult
+}
+
+"""
+Additional properties about a ERModelRelation
+"""
+type ERModelRelationEditableProperties {
+
+    """
+    Documentation of the ERModelRelation
+    """
+    description: String
+    """
+    Display name of the ERModelRelation
+    """
+    name: String
+}
+
+"""
+Additional properties about a ERModelRelation
+"""
+type ERModelRelationProperties {
+
+    """
+    The name of the ERModelRelation used in display
+    """
+    name: String!
+    """
+    The urn of datasetA
+    """
+    datasetA: Dataset!
+
+    """
+    The urn of datasetB
+    """
+    datasetB: Dataset!
+
```

Comment:
let's rename these to match the model to use `source` and `destination`

Reply:
The only problem is that it is orthogonal.  Anything can be source and anything can be destination.  You can look from either direction.  And we made a conscious decision to name it A and B.

Reply:
Modified to source and destination